integration-tests/api/__tests__/admin/product.js

```diff
@@ -36,7 +36,12 @@ describe("/admin/products", () => {
   beforeAll(async () => {
     const cwd = path.resolve(path.join(__dirname, "..", ".."))
     dbConnection = await initDb({ cwd })
-    medusaProcess = await setupServer({ cwd, verbose: false })
+    medusaProcess = await setupServer({
+      cwd,
+      env: {
+        CACHE_TTL: 0,
+      },
+    })
```

Comment:
suggestion(non-blocking): Why is this added? Think a comment here would be appropriate :)